LICENSE.md

```diff
@@ -0,0 +1,27 @@
+Copyright 2019-2021 by Felix Fontein <felix@fontein.de>.
+Parts copyrighted 2006-2017 by the Pygments team, see AUTHORS at
+https://github.com/pygments/pygments/blob/3e1b79c82d2df318f63f24984d875fd2a3400808/AUTHORS.
+Parts copyrighted by Norman Richards (original author of JSON lexer).
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are
+met:
+
+* Redistributions of source code must retain the above copyright
+  notice, this list of conditions and the following disclaimer.
+
+* Redistributions in binary form must reproduce the above copyright
+  notice, this list of conditions and the following disclaimer in the
+  documentation and/or other materials provided with the distribution.
```

Comment:
nit: here's a nicely formatted markdown template for BSD-2-clause license https://github.com/IQAndreas/markdown-licenses/blob/master/bsd-2.md — you may want to follow its formatting

Reply:
I made the formatting more similar.